### DISCUSSION TITLE
Connecting WooCommerce

### DISCUSSION THREAD

**Discussion by JustKenjyDev** [inventree/InvenTree#6157] [Q&A]
Hi all,

I've been testing InvenTree as a solution for our Inventory management. As I am no hero in development I thought I'd ask a question.

We want to sync the Inventree inventory with a WordPress(Woocommerce) website, or with a Shopify Website (I've seen a plugin for this, just need to set it up) as we are planning on selling some materials we use in installations (cameras for example)

I've been searching a lot but can't seem to find a KB or a 'tutorial' on how to sync WooCommerce and/Or Shopify. 

Is there anyone kind enough to explain it to me how to tackle this at best?

Thanks!

**Comment by matmair** [comment]
the shopify plugin has a [readme](https://github.com/invenhost/inventree-shopify) with install steps - not aware of any tutorials for setting it up, there wasn't much intrest so not much time was spend on it.

**Comment by Timieh** [comment]
Follow, gonna build a new shop. And also really interested in Inventree integration. 
don't like having multiple stock amount (locations) files.

**Reply by Timieh** [reply]
@JustKenjyDev any update ?

**Reply by JustKenjyDev** [reply]
I no longer work for said company sadly enough so dropped the project! I hope you can figure it out tho!
